### ISSUE TITLE
Changing VoiceFlow does not update the skill

### ISSUE THREAD

**Issue by elyonardo** [mogapog/voiceflow-alexa#1]
Hi!
Thank you very much to share this!
It works for me....or better to say that SOMETIMES it works.
If I create a project in voiceflow then a skill in Alexa following your tutorial it works, but if I change something in Voiceflow it seems like the Alexa skill isn't updating nor in test tab nor on my echo show.

How can I fix this, in your opinion?

**Comment by mogapog** [comment]
I couldn't really tell you why it happens, since that's an error in the voiceflow dialog api and it didn't occur to me but i'm going to try helping you with it:

- 1: Double-triple check that the api key is right 
- 2: Try running your voiceflow assistant on the voiceflow page before using it on your alexa
- 3: Wait 30 minutes or something after running your voiceflow assistant on voiceflow
- 4: **MOST IMPORTANT:** Try hitting the publish button on voiceflow before running. It wasn't necessary for me but they could've updated it so i recommend you try that out.

**Comment by elyonardo** [comment]
Hi!
I checked the api key and it is correct.
Then I tried to run the assistant on voiceflow and to publish it.
Surprisingly it works and on ASK and on Echo Show!

So to update the skill, once updated the voiceflow assistant, it is necessary to run the assistant on voiceflow and then click on publish.

That's so strange....but ok, I guess

Just one thing:
I suggest you to update your readme file add the delete of HelloWorldIntent, as you recommend on reddit, and also with the strange behavior that occurred to me.

Thank you so much for your help :-)
